robottelo/ui/rhai.py

```diff
@@ -27,23 +27,19 @@ def register_client_to_rhai(self, activation_key, org):
         self.vm.register_contenthost(activation_key, org)
 
         # Red Hat Access Insights requires RHEL 6/7 repo and it not
-        # possible to sync the repo during the tests,
-        # adding a file in /etc/yum.repos.d/rhel6/7.repo
+        # possible to sync the repo during the tests, Adding repo file.
 
-        rhel6_repo = conf.properties['insights.rhel6_repo']
+        rhel6_repo = conf.properties['clients.rhel6_repo']
+        self.vm.configure_rhel_repo(rhel6_repo)
 
-        repo_file = (
-            '[rhel6-rpms]\n'
-            'name=RHEL6\n'
-            'baseurl={0}\n'
-            'enabled=1\n'
-            .format(rhel6_repo)
-        )
+        try:
+            insights_repo = conf.properties['insights.insights_el6repo']
+            self.vm.run('wget -O /etc/yum.repos.d/insights.repo {0}'.format(
+                insights_repo))
+        except:
+            pass
 
-        self.vm.run(
-            'echo "{0}" >> /etc/yum.repos.d/rhel6.repo'
-            .format(repo_file)
-        )
+        self.hostname = self.vm.hostname
```

Comment:
why duplicating this? You already have this under the vm instance.... please drop this and use self.vm.hostname where needed.